### ISSUE TITLE
Input Data Type [hg38]

### ISSUE THREAD

**Issue by wendy321** [cBioPortal/cbioportal#2070]
Hello all,

Same as my previous question - I am not sure whether it is fine to ask the question here or not. If it's not, please let me know and suggest me where I can ask.

I found brca_tcga_data_cna_hg19.seg and brca_tcga_meta_cna_hg19_seg.txt in  /cbioportal/core/src/test/scripts/test_data/study_es_0 folder. I guess the tool doesn't support hg38. Is that correct? If I want to make the local version cBioPortal supports hg38, what should I do? Could anyone give some suggestions?

Thank you & Best Regards,

Wendy
Web Application Engineer I at QBRC Lab, UTSouthWestern Medical Center

**Comment by pieterlukasse** [comment]
Hi Wendy, actually there are very few places where some (limited) genome build information is used. I.e. there are only a few UIs that are assuming a specific build: the patient view which contains a genome-wide overview of the CNA and mutations for a patient and the new CN segments view (I'm less familiar with this one, it could be it already supports hg38). One more place is the validator, which enforces hg19 (but you could skip that). 

In a nutshell: with a little bit of effort, hg38 support could be added. @jjgao what do you think?

**Comment by wendy321** [comment]
Hi Pieterlukcasse,

Thank you for the replying. 

(1) For the **UIs** that **don't assume** a specific build, if I import hg38 into database, can we still view them?
For the **other UIs** that **assume** a specific build, do you know which files related to them? Maybe I can take a look at it and figure out how to modify them.

(2) As for the backend, I reviewed the data formats in the documentation. I guess ***.seg**, *****_seg.txt**, and other kinds of data with **reference_genome_id** field might be affected when changing to hg38. Is that correct? Are there any other data or tables of database should be modified?

Best Regards,

Shin-Yi
Web Application Engineer I at QBRC Lab, UTSouthWestern Medical Center
shinyi.lin@utsouthwestern.edu

**Comment by jjgao** [comment]
closing this. For further questions, please contact cbioportal@googlegroups.com
